src/main.lua

```diff
@@ -138,6 +138,25 @@ local function addRecordResultFields(ioEvent, result)
 	end
 end
 
+local function addAuctionResultFields(ioEvent, result)
+	ioEvent:addFieldsIfExist(result, {
+		"bidAmount",
+		"bidder",
```

Comment:
Will this always be the same as the from field? If so we can likely omit this one.

Reply:
it will, can remove